### PR TITLE
WebUI: close embedded panel when clicking prev/next

### PR DESCRIPTION
**Pull request by jelly**
When the Storage drawer is opened, automatically close it when clicking previous or next.

Resolves: INSTALLER-3405

### CODE REVIEW

File: ui/webui/test/check-progress [hunk 1]
```diff
@@ -31,11 +31,19 @@ class TestInstallationProgress(anacondalib.VirtInstallMachineCase):
...
 
         i.open()
+        drawer_open = False
 
         for page in i.steps[:-1]:
             # with the pages basically empty of common elements (as those are provided by the top-level installer widget)
             # we at least iterate over them to check this works as expected
             i.wait_current_page(page)
+            if drawer_open:
+                self.assertFalse(b.is_present(".pf-c-drawer__panel-main"))
```
Comment (rvykydal) [inline ui/webui/test/check-progress:41]: I think the common practice in our tests is to use the element ids via helper functions in test/helpers/*.py. For example check_ or click_ funcitons in test/helpers/storage.py. So maybe it is worth following it also here, like having check_drawer_open function somewhere?

**Review by M4rtinK — APPROVED** [review]
Looks good to me, thanks! :) 

Also agreed with Radek on ideally accessing this via the library function in tests. :)

File: ui/webui/test/check-progress (hunk 1 above)
Comment (jelly) [reply ui/webui/test/check-progress]: Added helper functions

**Review by rvykydal — APPROVED** [review]
Looks good to me.

**Review by VladimirSlavik — APPROVED** [review]
Looks good to me. Thank you!

**Pull request merged**